src/pages/videopage/videopage.js

```diff
@@ -24,6 +26,20 @@ export default function Videopage() {
             navigate("/login");
         }
     }
+    const toggleLike = () => {
+        if (liked) {
+            removeFromLikes();
+        } else {
+            addToLikes();
+        }
+    }
+    const toggleWatchLater = () => {
+        if (inWatchLater) {
+            removeFromWatchLater();
+        } else {
+            addToWatchLater();
+        }
+    }
     const handleCreatePlaylist = () => {
         dispatch({ type: "SHOW_MODAL" });
         navigate("/playlists");
```

Comment:
suggestion : I would be better if user is not navigate to playlist page ,it is not a good user experience . A toast could be show when a playlist is created successfully